test/cppsuite/tests/test_live_restore.cpp

```diff
@@ -311,72 +306,138 @@ get_stat(WT_CURSOR *cursor, int stat_field, int64_t *valuep)
+      source + "\"),cache_size=1GB," + verbose_string +
+      ",statistics=(all),statistics_log=(json,on_close,wait="
       "1)";
 
-    logger::log_msg(LOG_TRACE, "arg count: " + std::to_string(argc));
-    bool fresh_start = false;
-    bool background_debug = false;
-    if (argc > 1 && argv[1][1] == 'f') {
-        fresh_start = true;
-        logger::log_msg(LOG_WARN, "Started in -f mode will clean up existing directories");
-        // Live restore expects the source directory to exist.
-        testutil_recreate_dir(SOURCE_DIR);
-        testutil_remove("WT_TEST");
-    } else if (argc > 1 && argv[1][1] == 'b') {
-        /*
-         * Operate in background thread debug mode, in this case we turn off CRUD operations and
-         * just wait until the relevant statistic is set then exit.
-         *
-         * Right now background_debug and fresh_start are mutually exclusive.
-         */
-        background_debug = true;
-    }
-    // We will recreate this directory every time, on exit the contents in it will be moved to
-    // WT_LIVE_RESTORE_SOURCE/.
-    testutil_recreate_dir("WT_TEST");
+    //,debug=(fill_holes_on_close=true)
 
     /* Create connection. */
-    if (fresh_start)
-        connection_manager::instance().create(conn_config, DEFAULT_DIR);
+    if (first)
+        connection_manager::instance().create(conn_config, home);
     else
-        connection_manager::instance().reopen(conn_config, DEFAULT_DIR);
+        connection_manager::instance().reopen(conn_config, home);
 
     auto crud_session = connection_manager::instance().create_session();
+    if (!background_thread_mode)
+        do_random_crud(crud_session, collection_count, op_count, first);
 
-    if (!fresh_start && !background_debug)
-        configure_database(crud_session);
-
-    if (background_debug) {
-        // Loop until the state stat is complete!
+    // Loop until the state stat is complete!
+    if (thread_count > 0 && (background_thread_mode && !first)) {
         int64_t state = 0;
+        logger::log_msg(LOG_INFO, "Waiting for background data transfer to complete...");
         while (state != WT_LIVE_RESTORE_COMPLETE) {
             auto stat_cursor = crud_session.open_scoped_cursor("statistics:");
             get_stat(stat_cursor.get(), WT_STAT_CONN_LIVE_RESTORE_STATE, &state);
             __wt_sleep(1, 0);
         }
-    } else
-        do_random_crud(crud_session, fresh_start);
+        logger::log_msg(LOG_INFO, "Done!");
+    }
 
     // We need to close the session here because the connection close will close it out for us if we
     // don't. Then we'll crash because we'll double close a WT session.
     crud_session.close_session();
     connection_manager::instance().close();
-    testutil_remove(SOURCE_DIR);
-    testutil_copy("WT_TEST", SOURCE_DIR);
+}
+
+/* Usage:
+ *  -b background thread debug mode, initialize the database then loop for iteration count. Each
```

Comment:
I thought about a `-u` to print the usage message but how would you know it's `-u` given that no configurations being passed will result in the default test running?

Reply:
I could change the home path to `-H` and help to `-h`, thoughts?

Reply:
We should have an established pattern for this in the csuite tests. Top of my head it's `-?` for help messages

Reply:
ah this gets me all the time in our scripts/cli tools. We're not consistent.
## wt tool
`-?` for help
`-h` for home
## run.py
`-h` for help
`-D` `--dir` for home
## test format
`bad command` for help
`-h` for home
## cppsuite run
`-h` for help
`-H` for home

`run.py` makes the most sense to me, followed by `wt tool`. We should standardise these one day

Reply:
Given the cppsuite's own run function does -H i'll do that too. We should create a follow up ticket for the backlog to fix this. 😂

Reply:
Created https://jira.mongodb.org/browse/WT-13921

Reply:
Usage message example:
```
$ ./test/cppsuite/test_live_restore -h
Usage: ./test_live_restore [OPTIONS]
DESCRIPTION
         The live restore test simulates performing a live restore on a database. If no options are specified it will run with a default configuration.
OPTIONS
        -b Background thread debug mode, initialize the database then loop for iteration count. Each iteration will wait for the background thread to finish transferring data before terminating. No additional CRUD operations will take place during background thread debug mode.
        -c The maximum number of collections to run the test with, if unset collections are created at random.
        -H Specifies the database home directory.
        -h Output a usage message and exit.
        -i Iteration count, the number of iterations to run the test program. Note: A value of 1 with no source directory specified will simply populate a database i.e. no live restore will take place. The default iteration count is 2.
        -l Log level, this controls the level of logging that this test will run with. This is distinct from the verbose level option as that is a WiredTiger configuration. Default is LOG_ERROR (0). The other levels are LOG_WARN (1), LOG_INFO (2) and LOG_TRACE (3).
        -o Op count, op_count the number of crud operations to apply while live restoring.
        -t Thread count for the background thread. A value of 0 is legal in which case data files will not be transferred in the background.
        -o Verbose level, this setting will set WT_VERB_FILE_OPS with whatever level is provided. The default is off.
$ echo $?
1
```

Reply:
Implemented here: ee6ee767e